### PR TITLE
This PR rebrands mytunnel to fusetunnel 

### PR DESCRIPTION
**Pull request by matthewzhaocc**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Roman Shtylman
```
Comment (bennbollay) [inline LICENSE]: Ah, no, this probably needs to be kept as Roman.  It's still largely his copyright.

File: package.json [hunk 2]
```diff
@@ -1,18 +1,13 @@
...
-    "url": "git://github.com/StyleT/mytunnel.git"
+    "url": "git://github.com/fusebit/tunnel.git"
   },
-  "author": "Vladlen Fedosov <vlad.fedosov@gmail.com>",
-  "contributors": [
-    "Roman Shtylman <shtylman@gmail.com>",
-    "Gert Hengeveld <gert@hichroma.com>",
-    "Tom Coleman <tom@hichroma.com>"
-  ],
+  "author": "Fusebit",
```
Comment (bennbollay) [inline package.json]: Keep the contributors block, since we're derived.  And add yourself :D

**Review by bennbollay — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -1,18 +1,13 @@
 {
-  "name": "mytunnel",
+  "name": "fusetunnel",
```
Comment (bennbollay) [inline package.json]: Package should be @fusebit/tunnel (probably).

Comment (matthewzhaocc) [reply package.json]: Done

File: package.json [hunk 4]
```diff
@@ -1,18 +1,13 @@
...
-  "author": "Vladlen Fedosov <vlad.fedosov@gmail.com>",
-  "contributors": [
-    "Roman Shtylman <shtylman@gmail.com>",
-    "Gert Hengeveld <gert@hichroma.com>",
-    "Tom Coleman <tom@hichroma.com>"
-  ],
+  "author": "Fusebit",
   "main": "./localtunnel.js",
   "types": "./localtunnel.d.ts",
   "bin": {
```
Comment (bennbollay) [inline package.json:19]: The bin should probably go from `lt` to `ft`.

**Pull request merged**